src/wallet/rpcwallet.cpp

```diff
@@ -573,7 +573,7 @@ UniValue getreceivedbyaddress(const UniValue& params, bool fHelp)
     // Bitcoin address
     CBitcoinAddress address = CBitcoinAddress(params[0].get_str());
     if (!address.IsValid())
-        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid Bitcoin address");
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid Bitcoin address");
```

Comment:
Why not keeping `RPC_INVALID_ADDRESS_OR_KEY` and *adding* `RPC_INVALID_PARAMETER` (not renaming from `RPC_INVALID_PARAMETER`)? In this case (and the ones below) `RPC_INVALID_ADDRESS_OR_KEY` is more precise than `RPC_INVALID_PARAMETER`.